clang-tools-extra/clangd/unittests/CodeCompleteTests.cpp

```diff
@@ -4435,11 +4444,11 @@ TEST(CompletionTest, SkipExplicitObjectParameter) {
     auto Result = codeComplete(testPath(TU.Filename), Code.point("c1"),
                                Preamble.get(), Inputs, Opts);
 
-    EXPECT_THAT(Result.Completions,
-                UnorderedElementsAre(AllOf(named("foo"), signature("(int arg)"),
-                                           snippetSuffix("(${1:int arg})")),
-                                     AllOf(named("bar"), signature("(int arg)"),
-                                           snippetSuffix("(${1:int arg})"))));
+    EXPECT_THAT(
+        Result.Completions,
+        UnorderedElementsAre(
+            AllOf(named("foo"), signature("(int arg)"), snippetSuffix("")),
+            AllOf(named("bar"), signature("(int arg)"), snippetSuffix(""))));
```

Comment:
potential regression?

Reply:
Hah, it's from the removal of the `s` at the end of `a.$c1^s`! I guess there was a reason for it :laughing: 

I guess the reason is that now that we have a subsequent line that starts with `(`, the parser gets confused by the next token after the invocation point being `(`.

A semicolon (`;`) at the end of the line seems to do the trick as well, I suggest we go with that rather than an `s` (which confused me as a seemingly random character).